packages/launcher/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "launcher",
   "productName": "Axon",
   "version": "0.0.1",
+  "buildVersion": "0.0.1",
```

Comment:
Do we need to specify twice? Seems like some folks have used `process.env.npm_package_version` with success? Can we try that?

Reply:
no, this doesn't do anything. I tried the environment variable, but it does not exist in production. app.getVersion() works.